File: AD/HospitalCRUD/src/main/java/com/hospitalcrud/dao/model/Patient.java
package com.hospitalcrud.dao.model;

import com.hospitalcrud.dao.utilities.Constantes;
import lombok.Data;

import java.time.LocalDate;

@Data
public class Patient {
    private int id;
    private String name;
    private LocalDate birthDate;
    private String phone;
    private Credential credential;

    public Patient(int id, String name, LocalDate birthDate, String phone) {
        this.id = id;
        this.name = name;
        this.birthDate = birthDate;
        this.phone = phone;
    }

    public String toStringFichero() {
        return id + Constantes.SEPARADOR_CSV + name + Constantes.SEPARADOR_CSV
                + birthDate + Constantes.SEPARADOR_CSV + phone + "\n";
    }
}

File: Moviles/HospitalRoomCompose/app/src/main/java/com/example/hospitalroomcompose/domain/model/Patient.kt
package com.example.apptareascompose.domain.model

import java.time.LocalDate

data class Patient (
    val id : Int = 0,
    val name : String = "",
    val birthDate : LocalDate = LocalDate.now(),
    val phone : String = "",
    val paid : Int = 0,
)